src/panels/config/cloud/account/cloud-remote-pref.ts

```diff
@@ -49,39 +49,57 @@ export class CloudRemotePref extends LitElement {
 
     if (!remote_certificate) {
       return html`
-        <ha-card header="Remote Control">
+        <ha-card
+          header=${this.hass!.localize(
+            "ui.panel.config.cloud.account.remote.title"
+          )}
+        >
           <div class="preparing">
-            Remote access is being prepared. We will notify you when it's ready.
+            ${this.hass!.localize(
+              "ui.panel.config.cloud.account.remote.access_is_being_prepared"
+            )}
           </div>
         </ha-card>
       `;
     }
 
     return html`
-      <ha-card header="Remote Control">
+      <ha-card
+        header=${this.hass!.localize(
+          "ui.panel.config.cloud.account.remote.title"
+        )}
+      >
         <div class="switch">
           <ha-switch
             .checked="${remote_connected}"
             @change="${this._toggleChanged}"
           ></ha-switch>
         </div>
         <div class="card-content">
-          Home Assistant Cloud provides a secure remote connection to your
-          instance while away from home. Your instance
-          ${remote_connected ? "is" : "will be"} available at
+          ${this.hass!.localize(
+            "ui.panel.config.cloud.account.remote.info",
+            "is_will-be",
+            remote_connected ? "is" : "will_be"
+          )}
```

Comment:
Let's split those 2 sentences:
`Home Assistant Cloud provides a secure remote connection to your instance while away from home.`
And 2 different sentences:
`Your instance is available at {url}`
`Your instance will be available at {url}`

Reply:
Otherwise we would also have to translate "is" and "will be" and we don't know if they are in the same place in every language